.env.example

```diff
@@ -21,6 +21,8 @@ AWS_S3_BUCKET="action-XXXXXXXX"
 AWS_SECRET_ACCESS_KEY="XXXXXXXXXXXXXXXXXXXXXXXX"
 
 CDN_BASE_URL="//s3-host.parabol.co/static"
+DISABLE_CDN_BUILD=true                       # don't use the build from the CDN
```

Comment:
We could, I figured I'd just add it to the develop development environment we share internally on the team. I wasn't quite sure what script to add it to in `package.json` open to suggestions!